### ISSUE TITLE
Won't work on tabs

### ISSUE THREAD

**Issue by losewin** [mwean/sublime_jump_along_indent#11]

**Comment by mwean** [comment]
Yeah, you're right. I only use spaces, so I've never had a need to add tab functionality. There's a lot in `file_scanner.py` that's built for spaces, so we'd have to abstract that a bit. I probably won't be able to make the changes, but I'm happy to review a PR and help get things working.
